From e03225f3c6c77e7566bc5ee52ef701815c783084 Mon Sep 17 00:00:00 2001
From: Sachin Prabhu <sprabhu@redhat.com>
Date: Mon, 28 Jul 2014 12:16:55 -0400
Subject: [fs] cifs: Update cifs version number to 2.03

Message-id: <1406549822-10376-63-git-send-email-sprabhu@redhat.com>
Patchwork-id: 86735
O-Subject: [RHEL7.1 PATCH 62/69] [bz1082049]Update cifs version number to 2.03
Bugzilla: 1082049
RH-Acked-by: J. Bruce Fields <bfields@redhat.com>
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Steve French <smfrench@gmail.com>

Signed-off-by: Steve French <smfrench@gmail.com>
(cherry picked from commit 2e4b8c2c3b6247423c6d7097ab6b33e07cb624dc)
Signed-off-by: Sachin Prabhu <sprabhu@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/cifs/cifsfs.h b/fs/cifs/cifsfs.h
index 3648e6f..3d43b4e 100644
--- a/fs/cifs/cifsfs.h
+++ b/fs/cifs/cifsfs.h
@@ -140,5 +140,5 @@ extern long cifs_ioctl(struct file *filep, unsigned int cmd, unsigned long arg);
 extern const struct export_operations cifs_export_ops;
 #endif /* CONFIG_CIFS_NFSD_EXPORT */
 
-#define CIFS_VERSION   "2.02"
+#define CIFS_VERSION   "2.03"
 #endif    /* _CIFSFS_H */
-- 
1.7.1